src/sfx/sfx.cpp

```diff
@@ -0,0 +1,14 @@
+#include "sfx.hpp"
+#include "soloud.h"
+#include "en/ComponentManager.hpp"
+#include "en/World.hpp"
+
+namespace ari::en
```

Comment:
change the namespace to ari::sfx from ari::en::sfx
and write the name space like this
`namespace ari::sfx`

Reply:
ok sure.